### PR TITLE
Added features based on suggestion by Vespura

### PR DESCRIPTION
**Pull request by glitchdetector**
Suggestion: https://forum.fivem.net/t/release-server-password-make-your-server-secure-and-scum-free-1-0/180499/7?u=glitchdetector

Features:
Temp-ban system when a player is kicked for invalid password attempts (configurable time)
Time remaining shown in "ban" message when player tries to reconnect
Multiple attempts
Bypass ACE permission system

Bugfixes:
Fetching the ped every time we want to manipulate it to prevent invalid ped id being used

### CODE REVIEW

**Comment by glitchdetector** [comment]
(oh and it actually shows "attempts remaining" and "attempt remaining" based on the number of attempts remaining)

File: client.lua [hunk 1]
```diff
@@ -1,13 +1,15 @@
+-- PASSWORD STUFF
 ------------------------------------
 -- Server Password, Made by FAXES --
+--   Modified by GlitchDetector   --
```
Comment (Blumlaut) [inline client.lua:4]: them credit hijacking tho

Comment (FAXES) [reply client.lua:4]: haha

**Comment by glitchdetector** [comment]
Changes has been implemented manually instead of merging the pull request

**Pull request closed without merging**